Makefile

```diff
@@ -18,14 +18,15 @@ else
   $(error '$(ARCH)' is not a valid architecture, must be one of: RV32, RV64)
 endif
 
-SAIL_FLEN := riscv_flen_D.sail
+SAIL_FLEN := riscv_flen_Q.sail
```

Comment:
Would you suggest adding a command line option? I think it would need to be a compile-time option if so. Or do we leave the makefile as is and have people manually change SAIL_FLEN before compiling if they want f128?

This issue will ideally go away once configuration via riscv-config is integrated, but since that might not be for a while, we probably need a solution until then.

Reply:
Yeah I think currently the best option would be to make it a Make variable, so people have to do

```
ARCH=RV64 FLEN=128 make ...
```

I don't think we want to parse `riscv-config` at compile time. The thing that would really help this is https://github.com/rems-project/sail/issues/534 but there's no timeline for that so I think the Make variable is the way to go for now. We should have one for VLENMAX too really.

Reply:
When the Make variable is set to 64, it causes issues with the boxing/unboxing functions. Hopefully switching over to the generic version will solve this.

```
model/riscv_fdext_regs.sail:51.11-27:
51 |    128 if regval [127..16] ==  0x_FFFF_FFFF_FFFF_FFFF_FFFF_FFFF_FFFF
   |           ^--------------^ checking function argument has type bool('p)
   | Could not resolve quantifiers for subrange_bits
   | * (0 <= 16 & (16 <= 127 & 127 < 64))
```